### PR TITLE
[#40] Handle UnicodeEncodeError filenames for PUT

### PR DESCRIPTION
**Pull request by alanking**
Tests ran successfully against 4.2.5 installation.

### CODE REVIEW

**Comment by alanking** [comment]
I was trying to think of the best way to capture the remaining work. I left some TODOs in the test code, but was hoping for something more... discoverable. Were we thinking of releasing off of this, or very soon after?

**Comment by trel** [comment]
Not necessarily - we can create issues for outstanding work and milestones in this repo if needed.

No current known blockers or requirements for a next release, just want coverage and confidence in case someone upgrades/pulls that they're not going to go backwards.

**Comment by alanking** [comment]
TODOs annotated with issue #60 to capture remaining work

File: irods_capability_automated_ingest/test/test_irods_sync.py [hunk 1]
```diff
@@ -880,6 +884,110 @@ def test_pre_job(self):
...
+            self.assert_logical_path(session)
+            self.assert_physical_path_and_resource(session, expected_physical_path)
+            self.assert_data_object_contents(session)
+            self.assert_metadata_annotation(session)
+            self.assert_data_object_size(session)
+
+        self.do_assert_failed_queue(count=None)
+        self.do_assert_retry_queue(count=None)
+
+    # TODO: #60 - Add register_as_replica_sync test
```
Comment (trel) [inline irods_capability_automated_ingest/test/test_irods_sync.py]: these should point to 63?

**Pull request merged**